### ISSUE TITLE
Does devcontainer.json work with something that is not Docker?

### ISSUE THREAD

**Issue by PmegaOcalc** [allaboutapps/go-starter#150]
Docker makes my Macbook Pro (16-inch, 2019) fly away into the sky.

**Comment by majodev** [comment]
Hi,

TL;DR: `devcontainer.json` / VSCode Remote Containers does not work without a *Docker-like* daemon.

Several of my colleagues use your Macbook Pro model (with an i7/9, 32GB RAM) in combination with [Docker for Mac](https://docs.docker.com/desktop/mac/install/) without problems. Resource-utilization should be OK after the initial setup phase (booting up the containers, installing the Go tools within vscode, first run of `make all` within the container). Please use the latest [Docker for Mac](https://docs.docker.com/desktop/mac/install/) version (and eventually try a full [factory reset](https://docs.docker.com/desktop/mac/troubleshoot/) if you still encounter problems).

[Podman](https://www.redhat.com/sysadmin/replace-docker-podman-macos) may be an [useable](https://opensource.com/article/21/7/vs-code-remote-containers-podman) alternative to Docker for Mac already, but we haven't tried it yet and cannot guarantee it will work.

Please also note that we **do not officially support** pure local development environments. However, if you really want to try building the project locally, you would need to start something like [Postgres.app](https://postgresapp.com/), install the [Go toolchain](https://formulae.brew.sh/formula/go) locally, launch [integresql](https://github.com/allaboutapps/integresql) locally and change/export [the ENV settings](https://github.com/allaboutapps/go-starter/blob/master/docker-compose.yml#L31-L61). It's a tedious process, but this project should *unofficially* work without Docker if you manage to install all these base tools (and each tool within our [Dockerfile](https://github.com/allaboutapps/go-starter/blob/master/Dockerfile)).

Hope this helps,
Mario
